terraform/modules/mimir-ha/main.tf

```diff
@@ -0,0 +1,323 @@
+# TODO: Replace s3_integrator resource to use its remote terraform module once available
+resource "juju_application" "s3_integrator" {
+  count = var.create_s3_integrator ? 1 : 0
+  name = "s3-integrator"
+  model = var.model_name
+  trust = true
+
+  charm {
+    name    = "s3-integrator"
+    channel = var.channel
+  }
+  units = 1
+
+}
+
+module "mimir_coordinator" {
+  source     = "git::https://github.com/canonical/mimir-coordinator-k8s-operator//terraform?ref=feature/terraform"
```

Comment:
Is not expected to merge feature/terraform branch?

Reply:
It is, we are in the process of preparing the merger right now